packages/chain-adapters/src/evm/bnbsmartchain/BscChainAdapter.ts

```diff
@@ -32,6 +32,7 @@ export class ChainAdapter extends EvmBaseAdapter<KnownChainIds.BnbSmartChainMain
         assetId: bscAssetId,
         chainId: args.chainId ?? DEFAULT_CHAIN_ID,
         rpcUrl: args.rpcUrl,
+        api: args.providers.http,
```

Comment:
Might want to make this `unchainedHttpUrl` or something more descriptive to make it more obvious what this is?

Reply:
Going to leave as is and we can update later if it become confusing